src/components/formreporttype.jsx

```diff
@@ -0,0 +1,129 @@
+import React, { useState } from "react";
+import { Formik, Form, Field } from "formik";
+import Categories from "./services/categories.json";
+import * as Yup from "yup";
+import ErrorMsg from "./errormsg";
+
+const getSubCategories = (categories, categoryId) => {
+  var cats = categories.find(category => category.id === categoryId);
```

Comment:
this should be

```js
var category = ...
```

cats seem like it would be multiple categories.

Reply:
modified to category